### DISCUSSION TITLE
Nerd fonts icon not showing

### DISCUSSION THREAD

**Discussion by Yuxin0610** [NvChad/NvChad#1487] [Q&A] [answered]
I am using JetBrainsmono nerd fonts in iterm2, but it is not working in Vim.

Does anyone know how to resolve this issue?

![Screen Shot 2022-08-18 at 9 41 51 AM](https://user-images.githubusercontent.com/30561368/185449731-4d52ef99-1a03-44df-abd6-d5554c5aa773.png)

**Comment by siduck** [comment, marked as answer]
not a nvchad issue! its your terminal issue, try kitty , alacritty, wezterm

**Comment by barungh** [comment]
> I am using JetBrainsmono nerd fonts in iterm2, but it is not working in Vim.
> 
> Does anyone know how to resolve this issue?
> 
> ![Screen Shot 2022-08-18 at 9 41 51 AM](https://user-images.githubusercontent.com/30561368/185449731-4d52ef99-1a03-44df-abd6-d5554c5aa773.png)

Make sure you are using a nerd font

**Reply by Yuxin0610** [reply]
Yeah I am using a nerd fond in iterm2 like this:
![Screen Shot 2022-08-18 at 10 50 55 AM](https://user-images.githubusercontent.com/30561368/185461540-ea538bff-93df-450c-a2a8-0fa148a8e7b9.png)

Is there any change that I need to make in NvChad config?

**Reply by clatisus** [reply]
@saitama0610 try the font with _nerd font_ suffix in it's name

<img width="565" alt="image" src="https://user-images.githubusercontent.com/15963765/185462312-b3afad6a-910e-411f-9ad0-546c13b4845a.png">

**Reply by Yuxin0610** [reply]
This worked! Thank you so much!
